### DISCUSSION TITLE
[Feature request] Table with Column Toggle saved  Stateful

### DISCUSSION THREAD

**Discussion by MichelePalese** [primefaces/community#2254] [PrimeNG]
Hi,
If I implement toggleable columns in a table, it would be convenient if they were saved in storage with the stateful function. Also because if, for example, I deactivate a column and then change the size of another, the next time I load the modified size is set to a wrong column

**Comment by Joebeurg** [comment]
Hi, 

I finally found this discussion, as I opened an issue with details a few months ago: https://github.com/primefaces/primeng/issues/16711

I was able to get around it by creating a temporary local storage and force updating the PrimeNG local storage objects. 
Were you able to come up with a workaround?
